src/ancillary/anchor-request-params-parser.ts

```diff
@@ -96,7 +98,7 @@ export class AnchorRequestParamsParser {
     if (req.get('Content-Type') !== 'application/vnd.ipld.car') {
       // Legacy requests
       Metrics.count(METRIC_NAMES.CTRL_LEGACY_REQUESTED, 1)
```

Comment:
Are these still "legacy" 😅?

Reply:
yeah it's not, it's now back to being the main expected path.  I thought about renaming it but then it will break continuity with the existing metric name.  @gvelez17 curious your thoughts on if this is worth renaming now or not?

Reply:
hm I don't think i named this, i do not see `cas_server_ctrl_legacy_requested_total` on any of the primary cas dashboards (tho i didn't do a comprehensive search).  I think, since a number of metrics are in flux anyway, I would not worry about historical continuity right now, go ahead and rename it appropriately!

Reply:
renamed to `ctrl_json_requested`, which is a more descriptive and accurate parallel to `ctrl_car_requested`